index.bs

```diff
@@ -1,12 +1,13 @@
 <pre class=metadata>
-Title: Web Audio API
+Title: Web Audio API 1.1
 Shortname: webaudio
-Level: 1
+Level: 1.1
 Group: audiowg
-Status: ED
+Status: FPWD
```

Comment:
Shouldn't this stay `ED` for the main branch, and `FPWD` for a specific branch?

Reply:
I think when we discussed this matter in person, the resolution was to use the versioning to keep the latest REC with markups. (See https://github.com/WebAudio/web-audio-api/releases/tag/1.0-with-markups-tpac-2024)

Do we want to use separate branches to handle these? I think we should be able to specify the branch that we want to use for the GitHub landing, but we should discuss the logistics.

Reply:
It's more or less all branches (branches are just pointers to a commit in git). As long as the `main` branch is the `ED` and has the markup removed (so we can add more changes), and we have clear pointers to various steps we've done, I think we're good.

Reply:
I also prefer something simple, so we're aligned on that matter.

@svgeesus Should we use `ED` or `FPWD` for this meta entry?

Reply:
It is fine to use ED for clarity. I can change it to FPWD locally when publishing.

Reply:
Thanks! Done.